src/components/NcAppSettingsDialog/NcAppSettingsDialog.vue

```diff
@@ -145,6 +145,7 @@ export default {
 		@update:open="handleCloseModal">
 		<template #navigation="{ isCollapsed }">
 			<ul :aria-label="settingsNavigationAriaLabel"
+				v-show="!isCollapsed"
```

Comment:
Isnt v-if usually better?

Reply:
we can leave it for now with v-show

Reply:
In this case, it is fine, IMO. We just hide it on a small screen, not like having a different rendering branchs.

So, works like `display: none` in `@media` query =D